ONSdigital/dp-cantabular-csv-exporter
# upgrade dp-kafka

### What

- upgrade dp-kafka to latest tagged version, with data races fixed.
- use use confluentinc/cp-kafka:6.0.0 in component tests to match same image used in dp-compose, and same kafka version as in develop

### How to review

- Make sure component tests kafka/zookeeper images match dp-compose images
- Make sure code changes make sense
- Make sure unit/component tests pass

### Who can review

anyone (suggested @redhug1 )
